### DISCUSSION TITLE
Invoking a subprocess from the executable.

### DISCUSSION THREAD

**Discussion by sreekarreddy21** [pyinstaller/pyinstaller#6256] [Help]
Hi,

I am trying to execute a python CLI command as a subprocess (using PyQT5.Qprocess) from the GUI interface built with PyInstaller. The python scripts invoked by a CLI are added to the bundle as 'data files'. When I tried executing the application, it throws a "module not found error" as the application is invoking CLI from 'python.exe' in my local machine. How can I direct the application to invoke the CLI from a different location, where it can find the required modules that are bundled with PyInstaller? 

Thanks.

**Comment by bwoodsend** [comment]
See #5978.
